src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/HighHeapUsageOldGenRcaConfig.java

```diff
@@ -27,25 +27,33 @@
   private static final Logger LOG = LogManager.getLogger(HighHeapUsageOldGenRcaConfig.class);
   private int topK;
   public static final int DEFAULT_TOP_K = 3;
-  private static final String TOP_K_RCA_CONF = "top-k";
```

Comment:
Should we just call the class `OldGenRcaConfig` ?

Reply:
The reason I use this "HighHeapUsageOldGenRcaConfig" as the class name is because I want to keep it consistent with the actually RCA class name which is "HighHeapUsageOldGenRca".